packages/odyssey-storybook/src/components/odyssey-mui/Tooltip/Tooltip.stories.tsx

```diff
@@ -62,6 +67,19 @@ const storybookMeta: Meta<TooltipProps> = {
 
 export default storybookMeta;
 
+const showTooltip = async (
+  canvasElement: HTMLElement,
+  step: StepFunction<ReactRenderer, TooltipProps>,
+  action: string
+) => {
```

Comment:
Can we have this be an object? When I look at the function `showTooltip`, I don't know what any of these args are:
```
showTooltip(canvasElement, step, "Tooltip Default");
```

It's not clear what I would pass in and what these mean. Using an object means they're named properties, and I can understand them at a glance.

Reply:
Same with the other functions like `selectTab`.

Reply:
@KevinGhadyani-Okta The arguments are coming from the playwright's play function like in this example - https://storybook.js.org/docs/react/essentials/interactions#write-an-interaction-test

Instead of duplicating the code by adding a play funtion in every variant, I moved it to a common method which needs type information. I wonder this would be any helpful if I try to move it to an object/interface, as we still have to define the type for `step` and `canvasElement`. The last param is just a string to pass the variant info so we can see that during the CI runs.

Reply:
For sure. I'm asking if we can do it like this:

```ts
showTooltip({
  actionName: "Tooltip Default",
  canvasElement,
  step,
});
```